### PR TITLE
chore: wasmCloud 0.81 bumps

### PR DESCRIPTION
**Pull request by brooksmtownsend**
- chore: bump wasmcloud to 0.81
- chore: bump wash-lib to 0.16
- chore: bump wash-cli to 0.25

## Feature or Problem
This PR bumps wasmcloud, wash-lib, and wash-cli for release. We can merge this PR but it is not quite ready to release yet as there are one or two more changes required before we release:
- [x] Bump wasmCloud component adapters in wash-lib to wasmtime rc3 from @ricochet's repo
- [ ] Ensure that everything works out-of-the-box to build #1178, #1184, and our echo example
- [ ] ..?

## Release Information
wasmCloud 0.81
wash-lib 0.16
wash-cli 0.25

## Testing
I was able to `--dry-run` publish both wash-lib and wash-cli (minus the dependency on wash-lib 0.16).

Additionally, I used the following commands to verify the proper semver release for all of these crates:
```
➜ convco version --bump
0.81.0
➜ convco version --prefix wash-lib-v -P crates/wash-lib --bump
0.16.0
➜ convco version --prefix wash-cli-v -P crates/wash-cli --bump
0.25.0
```

### Acceptance or Integration
This PR changed the Windows artifact to `pc-windows-msvc` as per #807, so we're enforcing that the minimum version of wasmCloud is now going to be 0.81. This is beneficial as we can also have a certain level of confidence w.r.t the wit worlds that we are using.

### CODE REVIEW

**Review by connorsmith256 — APPROVED** [review]

**Comment by connorsmith256** [comment]
@brooksmtownsend I think the tests are failing because you bumped the version of the host, but it's not released yet

**Comment by brooksmtownsend** [comment]
> @brooksmtownsend I think the tests are failing because you bumped the version of the host, but it's not released yet

Oh yeah... that's another thing that we need to do 😅 let me break this into 2 PRs, one for the host and one for the washes

**Comment by brooksmtownsend** [comment]
@connorsmith256 actually I ended up pinning to 0.81-rc1 so we could do our testing with it. Included in a separate commit

File: crates/wash-cli/tests/common/mod.rs [hunk 1]
```diff
@@ -250,6 +250,8 @@ impl TestWashInstance {
         assert!(status.success());
 
         let out = read_to_string(&log_path).context("could not read output of wash up")?;
+        println!("wash up output: {}", out);
+        eprintln!("wash up output: {}", out);
```
Comment (connorsmith256) [inline crates/wash-cli/tests/common/mod.rs:254]: Merrrrrr

File: crates/wash-lib/src/start/wasmcloud.rs [hunk 2]
```diff
@@ -515,15 +512,15 @@ mod test {
 
     #[tokio::test]
     async fn can_properly_deny_elixir_release_hosts() -> anyhow::Result<()> {
```
Comment (connorsmith256) [inline crates/wash-lib/src/start/wasmcloud.rs:514]: `can_properly_deny_elixir_release_hosts` 🙃

File: crates/wash-cli/tests/common/mod.rs (hunk 1 above)
Comment (brooksmtownsend) [reply crates/wash-cli/tests/common/mod.rs:254]: Yeah this is from my temporary testing things, found out the test is failing on an `etxtfilebusy` on downloading wadm 0.9.1? I assume we're getting rate limited so I'm gonna re-run tomorrow.

**Review by rvolosatovs — APPROVED** [review]

**Pull request merged**